output/redis/README.md

```diff
@@ -23,6 +23,12 @@ gogstash output redis
 
 			// (optional), in seconds, default: 1
 			"reconnect_interval": 1
+			
+			// (optional) Password for DB, default: null
```

Comment:
The type of `password` is `string`, so I think the default value is an empty string `""`.